sdk/iOS/OoyalaSkinSDK/ViewControllers/FullscreenStateController/FullscreenStateController.m

```diff
@@ -11,14 +11,17 @@
 #import "PresentedViewControllerHelper.h"
 
 
-#define FULLSCREEN_ANIMATION_DURATION 0.5
+#pragma mark - Constants
+
+#define FULLSCREEN_ANIMATION_DURATION 0.4
```

Comment:
I think we better use static.

> Constants should be declared as static constants and not #defines unless explicitly being used as a macro

Reply:
It's old code, I just added pragma mark. We'll change code style after it will be finished.